src/components/PeopleAndRoles/OrgPerson.vue

```diff
@@ -297,20 +323,17 @@ export default class OrgPerson extends Mixins(CommonMixin) {
   }
 
   /**
-   * Called when component is created.
-   * Sets local properties if this an edit.
+   * Called when component is created to set local properties.
    */
   private created (): void {
     // safety check
     if (this.currentOrgPerson) {
-      this.orgPerson = { ...this.currentOrgPerson }
-      this.orgPerson.officer = { ...this.currentOrgPerson.officer }
-      // set the Director checkbox
-      this.isDirector = this.orgPerson.roles.some(party => party.roleType === Roles.DIRECTOR)
-      // set the Incorporator checkbox
-      this.isIncorporator = this.orgPerson.roles.some(party => party.roleType === Roles.INCORPORATOR)
-      // set the Completing Party checkbox
-      this.isCompletingParty = this.orgPerson.roles.some(party => party.roleType === Roles.COMPLETING_PARTY)
+      this.orgPerson = cloneDeep(this.currentOrgPerson)
```

Comment:
How does cloneDeep differ from say, Object.Assign?

Reply:
`Object.assign()` and `...object` both perform shallow copies -- they don't copy any sub-objects but instead ref the original objects.